bitcoin/src/sign_message.rs

```diff
@@ -126,6 +125,15 @@ mod message_signing {
             })
         }
 
+        /// Create from a byte slice.
+        #[deprecated(since = "0.32.0", note = "Use `from_byte_array` instead.")]
```

Comment:
```suggestion
        #[deprecated(since = "TBD", note = "Use `from_byte_array` instead.")]
```

The we set `TBD` right before release when we know the next release number (even though its almost certainly `0.33.0`).

Reply:
Done